java/client/src/org/openqa/selenium/chrome/ChromeDriver.java

```diff
@@ -179,6 +180,10 @@ public ChromeDriver(ChromeDriverService service, ChromeOptions options) {
   @Deprecated
   public ChromeDriver(ChromeDriverService service, Capabilities capabilities) {
     super(new ChromeDriverCommandExecutor(service), capabilities);
+    if (capabilities.getCapability("enableDownloading")!=null){
+      sendCommandForDownloadChromeHeadLess(
+          (String) capabilities.getCapability("enableDownloading"));
```

Comment:
`enableDownloading` is not a legal capability name. `chromium:enableDownloading` is. The reasoning is here: https://w3c.github.io/webdriver/webdriver-spec.html#dfn-extension-capability

Reply:
nit: We're not the Chromium project and shouldn't introduce capabilities in their namespace.

How about using the `selenium:chrome` prefix?